src/Promitor.Integrations.AzureMonitor/RequestHandlers/AzureResourceManagerThrottlingRequestHandler.cs

```diff
@@ -46,11 +54,19 @@ public AzureResourceManagerThrottlingRequestHandler(string tenantId, string subs
             _metricSinkWriter = metricSinkWriter;
             _metricsCollector = metricsCollector;
 
+            string id = applicationId;
+
+            if (authenticationMode == AuthenticationMode.ManagedIdentity)
+            {
+                id = string.IsNullOrEmpty(managedIdentityId) ? "system-assigned-identity" : managedIdentityId;
+            }
+
             _metricLabels = new Dictionary<string, string>
             {
                 {"tenant_id", tenantId},
                 {"subscription_id", subscriptionId},
-                {"app_id", applicationId}
+                {"authentication", authenticationMode.ToString()},
+                {"authentication_id", id},
```

Comment:
I think we should keep using `app_id`
```suggestion
                {"app_id", id},
```